server/schedule/scheduler.go

```diff
@@ -58,16 +52,18 @@ type Scheduler interface {
 	GetName() string
 	// GetType should in accordance with the name passing to schedule.RegisterScheduler()
 	GetType() string
-	GetInterval() time.Duration
+	GetMinInterval() time.Duration
+	GetNextInterval(interval time.Duration) time.Duration
 	GetResourceKind() core.ResourceKind
 	GetResourceLimit() uint64
 	Prepare(cluster Cluster) error
 	Cleanup(cluster Cluster)
 	Schedule(cluster Cluster) *Operator
+	IsAllowSchedule() bool
```

Comment:
I suppose we can remove `GetResourceKind()`, `GetResourceLimit()`, `IsAllowSchedule()` from interface, Schedulers can check limit by themselves.

Reply:
the controller should know `IsAllowSchedule`. Directly return nil operator will change tick time.